src/ngraph/frontend/onnx_import/op/quant_conv.cpp

```diff
@@ -166,8 +167,43 @@ namespace ngraph
                     int64_t groups{node.get_attribute_value<int64_t>("group", 1)};
 
                     auto data_scale = inputs.at(1);
```

Comment:
Let's leave the list of inputs on top of function please, followed by reading attributes.